resources/public/api/conf/1.0/application.raml

```diff
@@ -37,4 +37,4 @@ uses:
           description: VAT registration number. A nine-digit number.
           type: string
           example: 123456789
-      put: !include endpoints/sample_put.raml
\ No newline at end of file
+      put: !include endpoints/submit_VatReturn.raml
```

Comment:
this include doesn't work because the actual package is endpoints/insolventVAT. Considering this API only has one endpoint you can probably just move the submit_VatReturn.raml file to the resources/public/api/conf/1.0 folder (same as the application.raml file)